repository/src/main/java/org/apache/atlas/tasks/TaskRegistry.java

```diff
@@ -380,7 +390,12 @@ public List<AtlasTask> getTasksForReQueueIndexSearch() {
                         AtlasVertex vertex = iterator.next().getVertex();
 
                         if (vertex != null) {
-                            ret.add(toAtlasTask(vertex));
+                            AtlasTask atlasTask = toAtlasTask(vertex);
```

Comment:
Want to understand this once

Reply:
I've assigned it to a new AtlasTask object just for logging purposes